internal/acceptance/data_mlflow_model_test.go

```diff
@@ -0,0 +1,52 @@
+package acceptance
+
+import (
+	"testing"
+)
+
+func TestAccMlflowModel(t *testing.T) {
```

Comment:
There is already a test with the name `TestAccMlflowModel`, since this is for data source -- let's rename it to `TestAccDataMlflowModel`

```suggestion
func TestAccDataMlflowModel(t *testing.T) {
```